AgoraSocialApp/SvelteKit/src/routes/index.svelte

```diff
@@ -1,37 +1,12 @@
 <script>
-	import Counter from '$components/Counter.svelte';
-	//import {authenticate} from '$components/Auth/Auth.svelte';
+  import Home from '$components/Home.svelte';
+  import Login from '$components/Auth/Login.svelte';
+  import { isAuthed } from '$stores/session';
 </script>
-
-	<!-- <h1 use:authenticate>Hello world!</h1> -->
-
-	<Counter/>
-	<p>Visit the <a href="https://svelte.dev">svelte.dev</a> to learn how to build Svelte apps.</p>
-
-<style lang="scss">
-	h1 {
-		color: #ff3e00;
-		text-transform: uppercase;
-		font-size: 4rem;
-		font-weight: 100;
-		line-height: 1.1;
-		margin: 4rem auto;
-		max-width: 14rem;
-	}
-
-	p {
-		max-width: 14rem;
-		margin: 2rem auto;
-		line-height: 1.35;
-	}
-
-	@media (min-width: 480px) {
-		h1 {
-			max-width: none;
-		}
-
-		p {
-			max-width: none;
-		}
-	}
-</style>
\ No newline at end of file
+{#if $isAuthed !== null}
```

Comment:
honestly I would much prefer we redirect to /login and make index actually be index.  does `onMount`/`load` run before anything is rendered, perchance?

Reply:
agree, i was trying to figure out how the oidc subscription values were working, i'll move it out to a route

Reply:
updated to be like real work example in sveltekit, catches and goes to app or login. Protected all routes under /app for simplicity